elements/lisk-codec/src/codec.ts

```diff
@@ -113,6 +120,42 @@ export class Codec {
 		return (res as unknown) as T;
 	}
 
+	// For performance applications use decode() instead!
+	public decodeJSON = (schema: Schema, message: Buffer): GenericObject => {
```

Comment:
```suggestion
	public decodeJSON<T>(schema: Schema, message: Buffer): T {
```

Reply:
Had to make another adjustment so I committed from local repo. Done.